ed448/src/curve/twedwards/extensible.rs

```diff
@@ -78,12 +78,12 @@ impl ExtensiblePoint {
     pub fn add_extended(&self, other: &ExtendedPoint) -> ExtensiblePoint {
         let A = self.X * other.X;
         let B = self.Y * other.Y;
-        let C = self.T1 * self.T2 * other.T * FieldElement::TWISTED_D;
+        let C = self.T1 * self.T2 * other.T * FieldElement::EDWARDS_D;
         let D = self.Z * other.Z;
         let E = (self.X + self.Y) * (other.X + other.Y) - A - B;
         let F = D - C;
         let G = D + C;
-        let H = B + A;
+        let H = B - A;
```

Comment:
Again, this would be correct if `a` is `-1`, the linked papers says: `H = B - aA`.